pathfinding_service.rst

```diff
@@ -85,7 +84,13 @@ The arguments are POSTed as a JSON object.
 +----------------------+---------------+-----------------------------------------------------------------------+
 | max_paths            | int           | The maximum number of paths returned.                                 |
 +----------------------+---------------+-----------------------------------------------------------------------+
-| fee_iou              | object        | IOU object as described in :ref:`pfs_payment` to pay the service fee  |
+| iou                  | object        | IOU object as described in :ref:`pfs_payment` to pay the service fee  |
++----------------------+---------------+-----------------------------------------------------------------------+
+| diversity_penalty    | float         | (optional) Routing penalty per channel that is reused across multiple |
```

Comment:
Should we say something about the defaults here?

Reply:
I wouldn't until we have some real world experience in tuning those values, so that we don't have to update the specs every time we change a default value. Also we might want to vary the defaults within a certain range once we're able to gather routing feedback.